java/ql/test/library-tests/frameworks/spring/util/springutil.ssv

```diff
@@ -0,0 +1,144 @@
+org.springframework.util;AntPathMatcher;false;combine;;;Argument[0..1];ReturnValue;taint
```

Comment:
What is this file? Something committed by mistake?

Reply:
It's the file used to generate the original form of the tests.

Reply:
I've removed it since I didn't use it to regenerate test cases for the changes I made; not sure whether you wanted it in the first place.